wrappers/nodejs/binding.gyp

```diff
@@ -11,11 +11,26 @@
         "src/indy.cc"
       ],
       "link_settings": {
-        "libraries": [
-          "-L<(module_root_dir)",
-          "<!(node -e \"console.log((process.env.LD_LIBRARY_PATH || '').split(':').map(a => '-L' + a.trim()).filter(a => a != '-L').join(' '))\")",
-          "-lindy"
-        ]
+        "conditions": [
+          [ 'OS=="win"',
```

Comment:
I'm not familiar with GYP, could someone else double-check this changes?
btw I thought that it should be like
```
condtitions: [
  [ OS == win, { library_dirs, libraries } ]
  [ OS != win, { libraries } ]
]
```

Reply:
I have tested this on a MacBook and it works as intended. The section without the criteria is the default case. I've also seen examples online that use default criteria.

Reply:
Thanks for clarification